app/lib/menu/mac-os/mac-menu-builder.js

```diff
@@ -71,6 +71,16 @@ class MacMenuBuilder extends MenuBuilder {
     }));
   }
 
+  getWindowSubmenuTemplate() {
+    const submenuTemplate = super.getWindowSubmenuTemplate();
+
+    const fullScreenEntry = submenuTemplate.find(({ label }) => label === 'Fullscreen');
+
+    fullScreenEntry.accelerator = 'Ctrl+Cmd+F';
```

Comment:
Is this a macOS-only shortcut?

Reply:
Yup. And it's common among MacOS apps: VS Code, Chrome, Firefox, Finder, Preview etc.

Reply:
Do we already have for other platforms, like `F11` for Windows? Or do we already have it?